website/e2e-test/smoke-test.spec.js

```diff
@@ -32,3 +32,35 @@ const assertUrlLoads = async (page, url) => {
 test('Renders homepage', async ({ page }) => {
   await assertUrlLoads(page, EMBLEM_URL);
 });
+
+test('Renders login', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/login');
+});
+
+test('Renders logout', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/logout');
+});
+
+test('Renders donate', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/donate?campaign_id=59d32e9805fc4d3388db');
+});
+
+test('Renders viewCampaign', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/viewCampaign?campaign_id=59d32e9805fc4d3388db');
+});
+
+test('Renders createCampaign', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/createCampaign');
+});
+
+test('Renders viewDonation', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/viewDonation?donation_id=f5ea984abf29497bbed7');
+});
+
+// robots.txt - not working right now
```

Comment:
Add a link to [this issue](https://github.com/GoogleCloudPlatform/emblem/issues/389) in a comment?
```
// See https://github.com/GoogleCloudPlatform/emblem/issues/389
```

Reply:
Good idea, done.